src/main/java/com/mongodb/spark/sql/connector/config/MongoConfig.java

```diff
@@ -215,6 +215,27 @@ default WriteConfig toWriteConfig() {
     return writeConfig(getOriginals());
   }
 
+  /**
+   * Return a configuration consisting on options that begin with the namespace prefix.
```

Comment:
This Return header is already `@return` and should be removed.

Reply:
I like the docs to be readable top down (its mainly consumed in IDE). I have changed the text though.